src/OrleansTestingHost/TestCluster.cs

```diff
@@ -28,23 +28,43 @@ namespace Orleans.TestingHost
     /// </remarks>
     public class TestCluster
     {
+        /// <summary>
+        /// Primary silo handle
+        /// </summary>
```

Comment:
nit picking for future comments, as I know more are coming: if it's a simple short phrase, keep the comment and XML info in a single line, so that it doesn't unnecessarily occupy more than the actual code.